fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeViewModel.java

```diff
@@ -199,147 +166,44 @@ public void launchLao() {
-  }
-
-  public LiveData<SingleEvent<HomeViewAction>> getOpenConnectEvent() {
-    return mOpenConnectEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getOpenLaunchEvent() {
-    return mOpenLaunchEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getLaunchNewLaoEvent() {
-    return mLaunchNewLaoEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getCancelNewLaoEvent() {
-    return mCancelNewLaoEvent;
-  }
-
-  public Boolean isWalletSetUp() {
-    return mIsWalletSetUp.getValue();
-  }
-
   public LiveData<Boolean> getIsWalletSetUpEvent() {
     return mIsWalletSetUp;
   }
 
-  public LiveData<SingleEvent<Boolean>> getOpenWalletEvent() {
-    return mOpenWalletEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getOpenSeedEvent() {
-    return mOpenSeedEvent;
-  }
-
-  public LiveData<SingleEvent<String>> getOpenLaoWalletEvent() {
-    return mOpenLaoWalletEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getOpenSettingsEvent() {
-    return mOpenSettingsEvent;
-  }
-
-  public LiveData<SingleEvent<Boolean>> getOpenSocialMediaEvent() {
-    return mOpenSocialMediaEvent;
-  }
-
-  /*
-   * Methods that modify the state or post an Event to update the UI.
-   */
-
-  public void openLAO(String laoId) {
-    mOpenLaoEvent.postValue(new SingleEvent<>(laoId));
-  }
-
-  public void openHome() {
-    mOpenHomeEvent.postValue(new SingleEvent<>(true));
-  }
-
-  public void openConnecting(String laoId) {
-    mOpenConnectingEvent.postValue(new SingleEvent<>(laoId));
-  }
-
-  public void openWallet() {
-    mOpenWalletEvent.postValue(new SingleEvent<>(isWalletSetUp()));
-  }
-
-  public void openLaoWallet(String laoId) {
-    mOpenLaoWalletEvent.postValue(new SingleEvent<>(laoId));
-  }
-
-  public void openConnect() {
-    if (checkSelfPermission(getApplication().getApplicationContext(), Manifest.permission.CAMERA)
-        == PackageManager.PERMISSION_GRANTED) {
-      openQrCodeScanning();
-    } else {
-      openCameraPermission();
-    }
-  }
-
-  public void openQrCodeScanning() {
-    mOpenConnectEvent.setValue(new SingleEvent<>(HomeViewAction.SCAN));
-  }
-
-  public void openCameraPermission() {
-    mOpenConnectEvent.setValue(new SingleEvent<>(HomeViewAction.REQUEST_CAMERA_PERMISSION));
-  }
-
-  public void openLaunch() {
-    mOpenLaunchEvent.setValue(new SingleEvent<>(true));
-  }
-
-  public void openSettings() {
-    mOpenSettingsEvent.setValue(new SingleEvent<>(true));
-  }
-
-  public void openSocialMedia() {
-    mOpenSocialMediaEvent.setValue(new SingleEvent<>(true));
-  }
-
-  public void launchNewLao() {
-    mLaunchNewLaoEvent.setValue(new SingleEvent<>(true));
-  }
-
-  public void cancelNewLao() {
-    mCancelNewLaoEvent.setValue(new SingleEvent<>(true));
-  }
-
   public void setLaoName(String name) {
     this.mLaoName.setValue(name);
   }
 
-  public void setIsWalletSetUp(Boolean isSetUp) {
+  public void setIsWalletSetUp(boolean isSetUp) {
     this.mIsWalletSetUp.setValue(isSetUp);
   }
```

Comment:
I know this is not from you and not the point of the PR but would it not be better to use postValue ?

Reply:
Maybe, I am not certain of what the actual impact would be